scripts/dashboard/install-e2e-test-deps.sh

```diff
@@ -7,6 +7,29 @@ if [[ ! $(arch) =~ (i386|x86_64|amd64) ]]; then
 fi
 
 if grep -q  debian /etc/*-release; then
+    NODEJS_MAJOR_VERSION=14
+    DISTRO="$(lsb_release -cs)"
+    if [[ ! $(command -v node) || $(node --version | grep -oE "v([0-9])+" | cut -c 2-) < ${NODEJS_MAJOR_VERSION} ]]; then
+        sudo apt remove nodejs -y
```

Comment:
Will this remove all `node*` packages?

Reply:
no, i can do a purge and that might do. let me check

Reply:
done. and i think purge will take care of that. Oh and by node* packages you didn't mean the npm one right? just making sure if I understood you.